### PR TITLE
Ret 361

### PR DESCRIPTION
**Pull request by lvo700**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link ###

https://tools.hmcts.net/jira/browse/RET-361

### Change description ###

Added 'How would you like to be updated about your claim?' page and dummy pages for video hearings and claim has been saved.
When a user selects either email or post radio button and clicks 'Save and continue' button they will be redirected to the video hearings dummy page. If they select either radio button and click 'Save for later' they will be redirected to the claim has been saved dummy page. If the user attempts to click either 'Save for later' or 'Save and continue' button without selecting a radio button they will remain on the same page and an error message will be displayed.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x ] No
```

### CODE REVIEW

File: src/main/controllers/UpdatePreferenceController.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    });
+  }
+
+  public post(req: Request, res: Response): void {
+    if (req.body['update-preference'] === 'email' && req.body['saveButton'] === 'saveContinue') {
+      res.redirect('/would-you-want-to-take-part-in-video-hearings');
+    }
+    else if (req.body['update-preference'] === 'post' && req.body['saveButton'] === 'saveContinue') {
+      res.redirect('/would-you-want-to-take-part-in-video-hearings');
+    }
```
Comment (tautvydas-cerniauskas) [inline src/main/controllers/UpdatePreferenceController.ts:17]: Maybe we can merge these together? It also looks like it's something that could be separated into a separate function. Same for the other two `if` statements.

File: src/main/views/claim-saved.njk [hunk 2]
```diff
@@ -0,0 +1,17 @@
+{% extends "template.njk" %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+
+{% block beforeContent %}
+  {{ super() }}
+  {{ govukBackLink({
+  text: "Back",
```
Comment (tautvydas-cerniauskas) [inline src/main/views/claim-saved.njk:8]: This text will be taken from the common json once my PR goes in.

File: src/main/views/claim-saved.njk [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+  {{ super() }}
+  {{ govukBackLink({
+  text: "Back",
+  href: "/how-would-you-like-to-be-updated-about-your-claim"
+}) }}
+{% endblock %}
+{% block content %}
+  <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">{{ h1 }}</h1>
+  {{ govukPanel({
+  titleText: "Your claim has been saved"
```
Comment (tautvydas-cerniauskas) [inline src/main/views/claim-saved.njk:15]: Shouldn't this be taken from translation files?

File: src/main/views/disability.njk [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% block beforeContent %}
+  {{ super() }}
+  {{ govukBackLink({
+  text: "Back",
+  href: "/"
+}) }}
+{% endblock %}
+{% block content %}
+  <h1 class="govuk-heading-xl govuk-!-margin-bottom-2">Do you have a disability where you'd need assistance at a future tribunal hearing or to be sent information in an accessible format?</h1>
```
Comment (tautvydas-cerniauskas) [inline src/main/views/disability.njk:11]: All of this content should be taken from translation files.

File: src/main/views/claim-saved.njk (hunk 3 above)
Comment (lvo700) [reply src/main/views/claim-saved.njk]: Changed to template text.

File: src/main/controllers/UpdatePreferenceController.ts (hunk 1 above)
Comment (lvo700) [reply src/main/controllers/UpdatePreferenceController.ts]: Tidied up and replaced if-else with conditional operator.

**Review by marcjudgey — APPROVED** [review]

**Review by tautvydas-cerniauskas — APPROVED** [review]

**Pull request closed without merging**